public/css/concept-page.css

```
@@ -927,12 +977,35 @@
   color: var(--text-muted);
   font-size: 13px;
   font-weight: 650;
   line-height: 1.5;
 }
+.concept-page-b2__feedback-link {
+  display: inline;
+  min-height: 0;
+  margin: 0;
+  padding: 0;
+  border: 0;
+  border-bottom: 1px dotted currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)